assets/js/modules/analytics-4/components/audience-segmentation/settings/SettingsCardVisitorGroups/SetupSuccess/index.js

```diff
@@ -16,10 +16,15 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
```

Comment:
Nitpicky, but while we're at it:

```suggestion
 * WordPress dependencies
```